packages/clerk-js/src/ui/primitives/FormText.tsx

```diff
@@ -0,0 +1,26 @@
+import { forwardRef } from 'react';
```

Comment:
❓ What's the component about? The  <FormText/> naming is a bit generic. Should we maybe rename it to FormInfoText to complete the success, warning triplet?

Reply:
This displays a grey label without an icon. As you proposed `FormInfoText` seems to be a more appropriate name

Reply:
It's used for this
![image](https://user-images.githubusercontent.com/19269911/234859100-848b91c1-ab1f-4c49-81f9-efc01cb1ffac.png)